Initial commit of the openATTIC Manager module

LenzGr opened the pull request:
Create the directory and basic module initialization.

Add a `README.rst`

Signed-off-by: Lenz Grimmer <lgrimmer@suse.com>

rjfd commented:
@LenzGr the `__init__.py` you created should be renamed to `module.py`. Then you should add a file `__init__.py` with the following content:

```
from module import *  # NOQA
```

rjfd reviewed (changes requested):
https://github.com/openattic/ceph/pull/1#issuecomment-359410074

LenzGr commented:
@rjfd OK, will change this accordingly and rebase my PR

LenzGr committed:
Initial commit of the openATTIC Manager module

Create the directory and basic module initialization.

Add a minimal `README.rst`

Signed-off-by: Lenz Grimmer <lgrimmer@suse.com>

LenzGr commented on src/pybind/mgr/openattic/module.py:
```diff
@@ -0,0 +1,69 @@
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(Module, self).__init__(*args, **kwargs)
```
I currently get a `W0235:Useless super delegation in method '__init__'` - not sure if that can be ignored?

rjfd replied:
yes, you can ignore this. We need to be able to make pylint to know the existence of `mgr_module.py` file where the `MgrModule` class is declared.

ricardoasmarques reviewed (approved):
It works

rjfd reviewed (approved):
lgtm
